DNN Platform/Library/Common/Globals.cs

```diff
@@ -2992,9 +2992,9 @@ public static string GetRoleName(int roleId)
             switch (Convert.ToString(roleId))
             {
                 case glbRoleAllUsers:
-                    return glbRoleAllUsersName;
+                    return Localization.GetString("AllUsers.Role");

```

Comment:
I'm not sure `Globals.GetRoleName` should be returning localized role names. But if it should, the same change needs to be made lower in the method.

Reply:
Good catch. Thanks.